hedyorg/hedy
# [CHORE] Github Codespaces Setup

**Description**

Adds a `.devcontainer` folder with a setup for Github Codespaces (and VS Code Remote Container). Getting started with Hedy development has never been easier.

_Changes in detail. For example: "**Adds** translations of levels 1 to 12 to Polish" (note the use of present tense!)_

**Fixes _issue or discussion number_**

**How to test**

Open the following [link](https://github.com/codespaces/new?machine=basicLinux32gb&repo=599737621&ref=codespaces&location=WestEurope&devcontainer_path=.devcontainer%2Fdevcontainer.json) to start a Github Codespace with this setup.

After launching the codespace, an image will be build. When the image is build, a complete development environment for Hedy is shown. It is possible to start the app with `python app.py` and even run (and show) the cypress tests.

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
